scripts/publish_all.sh

```diff
@@ -1,29 +1,10 @@
 #!/usr/bin/env bash
 
-cd sdks/azure_core
-cargo publish
-cd ..
-
-sleep 20
-
-cd azure_storage
-cargo publish
-cd ..
-
-sleep 20
-
-cd azure_identity
-cargo publish
-cd ..
-
-cd azure_cosmos
-cargo publish
-cd ..
-
-cd azure_service_bus
-cargo publish
-cd ..
-
-cd azure_key_vault
-cargo publish
-cd ..
\ No newline at end of file
+for f in sdk/*
+do
+	echo "Publishing $f"
```

Comment:
nit: tabs or spaces

Reply:
What do you mean?

Reply:
It doesn't line up with the other commands because this line is using a tab and the others are spaces.